session_5/readme/README.md

```diff
@@ -0,0 +1,159 @@
+SCHEMA
+```ruby
+ActiveRecord::Schema.define(version: 2021_01_30_150003) do
+
+  create_table "assignments", force: :cascade do |t|
+    t.string "name"
+    t.decimal "weightage"
+    t.datetime "deadline"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "course_id"
+    t.index ["course_id"], name: "index_assignments_on_course_id"
+  end
+
+  create_table "branches", force: :cascade do |t|
+    t.string "name"
+    t.string "code"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "course_id"
+    t.integer "student_id"
+    t.index ["course_id"], name: "index_branches_on_course_id"
+    t.index ["student_id"], name: "index_branches_on_student_id"
+  end
+
+  create_table "courses", force: :cascade do |t|
+    t.string "name"
+    t.string "code"
+    t.integer "year"
+    t.integer "credits"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.integer "branch_id"
```

Comment:
Does that mean that only students of a particular branch can apply for courses offered by that branch?